torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2056,6 +2057,24 @@ def cast_symbool_to_symint_guardless(
     "IsNonOverlappingAndDenseIndicator": eval_is_non_overlapping_and_dense,
     "cast_symbool_to_symint_guardless": cast_symbool_to_symint_guardless,
     "math": math,
+    "RoundToInt": builtins.round,
+    "RoundDecimal": builtins.round,
+    "TruncToInt": math.trunc,
+    "IntTrueDiv": operator.truediv,
+    "FloatTrueDiv": operator.truediv,
+    "ToFloat": builtins.float,
+    "OpaqueUnaryFn_cos": math.cos,
+    "OpaqueUnaryFn_cosh": math.cosh,
+    "OpaqueUnaryFn_acos": math.acos,
+    "OpaqueUnaryFn_sin": math.sin,
+    "OpaqueUnaryFn_sinh": math.sinh,
+    "OpaqueUnaryFn_asin": math.asin,
+    "OpaqueUnaryFn_tan": math.tan,
+    "OpaqueUnaryFn_tanh": math.tanh,
+    "OpaqueUnaryFn_atan": math.atan,
+    "OpaqueUnaryFn_sqrt": math.sqrt,
+    "BitwiseFn_bitwise_and": operator.and_,
+    "BitwiseFn_bitwise_or": operator.or_,
```

Comment:
Do we need to add these back in, since https://github.com/pytorch/pytorch/pull/140597 removed them? Also, BitwiseFn_bitwise_and/or are no longer defined.

Reply:
No we don't